src/mobile/apps/artwork/components/meta_data/templates/inquiry.jade

```diff
@@ -1,13 +1,9 @@
-- var isAuction = artwork.auction && artwork.auction.is_auction
-- var newBuyNowFlow = sd.ENABLE_NEW_BUY_NOW_FLOW || (user && user.hasLabFeature("New Buy Now Flow"))
 - var buyable = artwork.is_buy_nowable || artwork.is_acquireable
```

Comment:
What is the difference now between `artwork.is_buy_nowable` and `artwork.is_acquireable` ?

Reply:
Agh, good catch.

`is_buy_nowable` checks that the artwork is `acquireable` and in an open auction (https://github.com/artsy/metaphysics/blob/031388753c43734ad89be8c8c267128d32cde12c/src/schema/artwork/index.ts#L318-L320)
`is_acquireable` just checks `acquireable`.

I'm not totally sure why we were checking both here, since the `is_acquireable` check should be enough.